ldapauth.go

```diff
@@ -544,3 +566,19 @@ func LogConfigParams(config *Config) {
 		LoggerDEBUG.Printf(fmt.Sprint(typeOfS.Field(i).Name, " => '", v.Field(i).Interface(), "'"))
 	}
 }
+
+// retrieve a secret value from environment variable or secret on the FS
+func getSecret(label string) string {
+	bindPassword := os.Getenv(strings.ToUpper(label))
+
+	if bindPassword != "" {
+		return bindPassword
+	}
+
+	b, err := os.ReadFile(fmt.Sprintf("/run/secrets/%s", strings.ToLower(label)))
```

Comment:
I am wondering if maybe we should allow a file path there, just because that would not be usable on Windows. Furthermore, Docker Swarm and Docker Compose allow mounting secrets into other locations than `/run/secrets`

Reply:
Support for [multiple paths](https://docs.docker.com/engine/swarm/secrets/) could be a problem. We could detect the `OS` system and define the full path based on default values o direct read the file if the label starts with `/` and exists. But I prefer to maintain the plugin as simple as possible.